### PR TITLE
5 types tests

### PR DESCRIPTION
**Pull request by laurayuwen**

### CODE REVIEW

**Comment by laurayuwen** [comment]
Jenkins: test this please.

**Comment by laurayuwen** [comment]
Jenkins: test this please.

**Comment by laurayuwen** [comment]
Jenkins: test this please.

**Comment by laurayuwen** [comment]
Jenkins: test this please.

**Comment by laurayuwen** [comment]
No idea why it is failing tests, works locally, not stable online.

**Comment by grosu** [comment]
So it looks like this was not merged, so I postponed working on 5_types this morning.

@laurayuwen:

1) Are you absolutely sure that you are running the same (latest) version of K locally?

2) Did you try to run those tests that failed, or some smaller one. using exhaustive search to make sure the non-determinism does not come from the semantics itself?

**Comment by laurayuwen** [comment]
Jenkins: test this please.

**Comment by andreistefanescu** [comment]
@grosu please review

File: k-distribution/tutorial/1_k/5_types/lesson_7/tests/config.xml [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 
   <include file="../../lesson_5/tests/config.xml"
            more-programs="."
-           more-results="." />
+           more-results="." >
+    <all-programs>
+      <krun-option name="--symbolic-execution" />
+      <krun-option name="--search" />
+      <!-- the pattern T:Type matches iff the program is well-typed -->
+      <krun-option name="--pattern" value="&lt;k&gt; T:Type &lt;/k&gt;" />
```
Comment (grosu) [inline k-distribution/tutorial/1_k/5_types/lesson_7/tests/config.xml:12]: Dirty but cool trick :)

**Pull request merged**